src/app/(main)/profile/reservations/page.tsx

```
@@ -28,35 +28,102 @@ interface reservationsType {
   userId: number;
 }
 
+type ReservationResponse = {
```

ESLint

[error] 'ReservationResponse' is defined but never used.

(@typescript-eslint/no-unused-vars)